### DISCUSSION TITLE
speeding up simulations

### DISCUSSION THREAD

**Discussion by Rinus-Verdonschot-UB** [asreview/asreview#1895] [Q&A]
Dear forum,

I have a question. How would one speed up ASReview simulations? Does ASReview automatically make use of all available cores? can it make use of the GPU (e.g. for more complex models like RF etc)? If one has access to a computing cluster how can one make use of the extra power provided?

Kind regards,

Rinus Verdonschot

**Comment by Rensvandeschoot** [comment]
You might be interested in this paper on [Optimizing ASReview Simulations: A generic Multiprocessing Solution for ‘Light-data’ and ‘Heavy-data’ Users](https://direct.mit.edu/dint/article/6/2/320/119106/Optimizing-ASReview-Simulations-A-generic)

**Reply by Rinus-Verdonschot-UB** [reply]
Thank you Rens, I will read this, if I have any questions, I'll get back to you.
